api/types/role.go

```diff
@@ -1586,6 +1589,19 @@ func validateRoleSpecKubeResources(version string, spec RoleSpecV6) error {
 	return nil
 }
 
+// setDefaultKubernetesVerbs sets the default verbs for each KubernetesResource
+// entry if none are specified. This is necessary for backwards compatibility
+// with older versions of Role: V3, V4, V5, and v6.
+func setDefaultKubernetesVerbs(spec *RoleSpecV6) {
+	for _, kubeResources := range [][]KubernetesResource{spec.Allow.KubernetesResources, spec.Deny.KubernetesResources} {
```

Comment:
Because we can deny some verbs for a certain pod:

```yaml

deny:
  kubernetes_resources:
  - name: undeletable-pod
    namespace: namespace
    verbs: ["delete"]
``` 

If it's an older role version which defined some `kubernetes_resources`, we need to wildcard the verbs